pkg/api/controller.go

```diff
@@ -726,7 +726,7 @@ func (c *Controller) GetPhysicalAddress(w http.ResponseWriter, r *http.Request,
 			StorageNamespace: repo.StorageNamespace,
 			Identifier:       address,
 			IdentifierType:   block.IdentifierTypeRelative,
-		}, block.PreSignModeWrite)
+		}, block.PreSignModeWrite, params.Path)
```

Comment:
Why do we need it in GetPhysicalAddress?
The user uses it to upload the file to path not to download

Reply:
Fixed - thanks!